src/components/app/routes/CourseRoute.jsx

```diff
@@ -1,21 +1,25 @@
+import { useParams } from 'react-router-dom';
 import { Container } from '@openedx/paragon';
 
 import {
   useCourseMetadata,
   useCourseRedemptionEligibility,
   useEnterpriseCourseEnrollments,
+  useEnterpriseCustomerContainsContent,
   useUserEntitlements,
 } from '../data';
 import NotFoundPage from '../../NotFoundPage';
 
 const CourseRoute = () => {
+  const { courseKey } = useParams();
```

Comment:
[inform] This file will soon be removed in Course migration. Coverage not important.